schemas/context/viewability-details.schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/viewability-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Viewability Details",
+  "type": "object",
+  "description": "Browser or app settings measured at runtime describing the viewability of an advertisement or content associated with each page load.",
+  "definitions": {
+    "viewabilityDetails": {
+      "properties": {
+        "xdm:adUnitDepth": {
+          "title": "Ad Unit Depth",
+          "type": "integer",
+          "description":
+          "Publishers may embed ad units inside containers/iFrames in order limit the ad's access solely to the code of the page. This describes how many 'containers' or iFrames the ad unit is displayed inside of.",
```

Comment:
What about ads used in other contexts than the web?

Reply:
MRC only defines this on web based ads. At the moment, mobile apps/ott/ctv/etc are not eligible for viewability measurements/standard.

Reply:
So then if this whole schema is based on this MRC work - let's make it clear in the description right up front (including the link to the spec).  I am tempted to also want it in the name - but that's overkill.

Reply:
Saw the new description file - VERY helpful, thanks!